flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_OFPData.js

```diff
@@ -417,69 +332,15 @@ class CDUAocOfpData {
     }
 }
 
-function setEstimatedBaggagePayload(mcdu) {
-    if (mcdu.aocWeight.fwdBag !== undefined || mcdu.aocWeight.rearBag !== undefined) {
-        return;
-    }
-
-    const currentPayload = getCurrentPayload(mcdu);
-
-    if (isNaN(currentPayload)) {
-        return;
-    }
-
-    const {
-        forwardBaggageWeight,
-        rearBaggageWeight
-    } = getDistributedBaggagePayload(currentPayload, mcdu.aocWeight.zfwcg);
-
-    mcdu.aocWeight.estFwdBag = forwardBaggageWeight;
-    mcdu.aocWeight.estRearBag = rearBaggageWeight;
-}
-
-async function loadBaggagePayload(mcdu, updateView) {
-    const FORWARD_BAGGAGE_INDEX = 3 + 1; // Value from flight_model.cfg
-    const REAR_BAGGAGE_INDEX = 5 + 1; // Value from flight_model.cfg
-
-    const currentfwdBag = getCurrentFwdBag(mcdu);
-    const currentRearBag = getCurrentRearBag(mcdu);
-
-    mcdu.aocWeight.loading = true;
-    updateView();
-
-    const payloadCount = SimVar.GetSimVarValue("PAYLOAD STATION COUNT", "number");
-    for (let i = 1; i <= payloadCount; i++) {
-        switch (i) {
-            case FORWARD_BAGGAGE_INDEX: {
-                await SimVar.SetSimVarValue(`PAYLOAD STATION WEIGHT:${i}`, "kilograms", +currentfwdBag);
-                break;
-            }
-            case REAR_BAGGAGE_INDEX: {
-                await SimVar.SetSimVarValue(`PAYLOAD STATION WEIGHT:${i}`, "kilograms", +currentRearBag);
-                break;
-            }
-            default: {
-                await SimVar.SetSimVarValue(`PAYLOAD STATION WEIGHT:${i}`, "kilograms", 0);
-                break;
-            }
-        }
-    }
-
-    mcdu.aocWeight.loading = false;
-    updateView();
-
-    return;
-}
-
 async function loadFuel(mcdu, updateView) {
     const currentBlockFuel = mcdu.aocWeight.blockFuel || mcdu.simbrief.blockFuel;
 
     mcdu.aocWeight.loading = true;
     updateView();
 
-    const outerTankCapacity = 228; // Left and Right // Value from flight_model.cfg
-    const innerTankCapacity = 1816; // Left and Right // Value from flight_model.cfg
-    const centerTankCapacity = 2179; // Center // Value from flight_model.cfg
+    const outerTankCapacity = 228 + (1 * 2); // Left and Right // Value from flight_model.cfg (plus the unusable fuel capacity (GALLONS))
```

Comment:
Just noticed that this is still in here. My previous comment is still applicable. Although the MSFS unusable fuel bug has been fixed so that we can now properly account for (that is, include) unusable fuel, we haven't done that yet because it is not compatible with our current engine model. Taz is working on updating both the fuel system and the engine model so that we can include unusable fuel. If you include it here now, the tank capacities in the EFB will not be the same as are used by the flight model and shown in the MSFS weight and balance UI. I recommend removing the unusable fuel quantities from here until Taz's work if finished.

Reply:
Noted. Will fix 👍🏻